CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # edge-react-gui
 
+## 2.0.14 (2021-06-24)
```

Comment:
Missing:
- Logs and CSV Export TX's show's what device was used to send the transaction